webapp/config/themeConfig/themeCreator.js

```diff
@@ -415,6 +415,35 @@ const themeCreator = (_themeVariables = {}, _themeConfig = {}) => {
       }
     },
 
+    // tx table row
+    txTableRow: {
```

Comment:
Does this table really need it's own special class and styles? My impression from the designs is that it's just using "universal" table stylings. Looking at the current table as it's rendered in the recent blocks plugin with the default skin (i.e. not the bMenace one that comes in the genesis theme) it looks pretty close to what it should for the tx list designs already (see screenshots below) aside from some aesthetic cleanup and solid background. I may be missing something, however, even if there are differences, it feels like this is something that should be generalized. I think we should have these classes be, as much as possible, something that would make sense to apply to other categories of tables and not just tx tables, so thinking of the themeCreator more in terms of categories rather than specific components, which makes it more flexible long term. My feeling looking at the tx table is that most of these should be available to any general table that has a lot of extra information it wants to display in an expanded row section. 

From recent blocks:
<img width="1173" alt="screen shot 2018-08-15 at 9 23 17 pm" src="https://user-images.githubusercontent.com/4344978/44188471-8cc38400-a0d2-11e8-9cca-f296d904247a.png">

From designs:
![screen shot 2018-08-06 at 1 01 03 pm](https://user-images.githubusercontent.com/4344978/44188478-9fd65400-a0d2-11e8-83e1-ccf53e2a4109.png)

Reply:
The current expand row is not flexible enough to make everything look exactly like our designs. It would be nice to have a component that just works by passing it in data, but I was shooting to go more similar to our design. I think creating something that works for everything will end up working for nothing

Reply:
I think that problem goes both ways though. Having components for every narrow use case will mean a ton of a components in the bpanel-ui library and themeConfigs here that are only used once. My feeling is that for any visual component that is very narrow and specific, it should just be built local to the plugin itself. Same for the themeConfigs. Components in bpanel-ui should ideally be as flexible as possible. I think this is even more important for the themConfig otherwise we just end up creating this gigantic, difficult to maintain object of very specific styles. For example, it's more useful to have something like `themeColors.primary` and `themeColors.secondary` that can be used throughout plugins and components rather than `header1Color`, `tableBorderColor`, etc. I don't see `txTableRow` being used anywhere except on our transaction table row. 

Updating the ExpandedDataRow to support something closer to what we want in the transaction table, e.g. having secondary data automatically split into two columns and supporting a footer where custom children components could be inserted wouldn't break the existing tables that are being used (in fact it would probably improve them and make them closer to the designs). Finally, for any stylings that specifically need to be used on just the transaction table rows, there's no reason why these styles couldn't be created in a style object local to the component right? There are a couple theme variables you would need to import and `makeGutter` is a utility already available in bpanel-ui.